apps/desktop/src/lib/project/project.ts

```diff
@@ -0,0 +1,36 @@
+export type KeyType = 'gitCredentialsHelper' | 'local' | 'systemExecutable';
```

Comment:
So this looks good in terms of grouping by feature, but what do you think about making these `../project/project.ts` type files `../project/index.ts` so that the import would be like:

```diff
- import { Project } from '$lib/project/project';
+	import { Project } from '$lib/project'; 
```

Reply:
I agree that we should improve on that. Let me still land this now and I'll continue with more stuff in a new PR.